tests/phpunit/bootstrap.php

```diff
@@ -13,7 +13,8 @@
 }
 
 define( 'BEANS_TESTS_DIR', __DIR__ );
-define( 'BEANS_TESTS_LIB_DIR', dirname( dirname( __DIR__ ) ) . '/lib/' );
+define( 'BEANS_ROOT_DIR', dirname( dirname( __DIR__ ) ) . DIRECTORY_SEPARATOR );
+define( 'BEANS_TESTS_LIB_DIR', BEANS_ROOT_DIR . 'lib' . DIRECTORY_SEPARATOR );
```

Comment:
All test directory constants will use the `DIRECTORY_SEPARATOR` to ensure tests work on both Windows and Mac environments.